src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -18,5 +20,7 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 	public void actionPerformed(ActionEvent e) {
 		if(e.getActionCommand().equals("Figure Joe 1")) FiguresJoe.figureScript1(driverManager.getCurrentDriver());
 		else if(e.getActionCommand().equals("Figure Joe 2")) FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+		else if(e.getActionCommand().equals("Square")) new SquareFactory(driverManager.getCurrentDriver()).createComplexCommand().execute();
+		else if(e.getActionCommand().equals("Triangle")) new TriangleFactory(driverManager.getCurrentDriver()).createComplexCommand().execute();
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.
You could also avoid duplication of "driverManager.getCurrentDriver())"